config/kn-event.yaml

```diff
@@ -0,0 +1,50 @@
+---
+config:
+  branches:
+    release-1.11:
+      openShiftVersions:
+        - skipCron: true
+          version: '4.15'
+    release-1.13:
+      openShiftVersions:
+        - version: '4.15'
+    release-1.14:
+      openShiftVersions:
+        - version: '4.15'
+    release-next:
+      openShiftVersions:
+        - version: '4.15'
+repositories:
+  - org: openshift-knative
+    repo: kn-plugin-event
+    imagePrefix: client-plugin-event
```

Comment:
It's a bit nit picking, but I wonder how the final names is generated from all those values. Is it going too verbose? I'd keep `kn-plugin-event` here. Wdyt?

Reply:
I updated the diff above. I think it's acceptable to proceed.

The naming thing. Yes, we could proceed with an image prefix of `kn-plugin-event`, however:

 - historically, the images were published under the prefix `client-plugin-event`, so keeping it makes the history consistent
 - we already burned down the naming convention in https://github.com/openshift-knative/kn-plugin-event/pull/442 (and its backports)

So, we can change the image prefix to `kn-plugin-event`, but I'm a bit reluctant due to the above points.